services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
To improve the code's readability and maintainability, it’s better to use the previously created function.
Reuse `get_movie_session_by_id` method here.